jekyll/_cci2/runner-feature-matrix.adoc

```diff
@@ -0,0 +1,121 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4+
+---
+= Self-hosted runner feature comparison
+:description: A comparison table of the features available for CircleCI runners.
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:experimental:
+
+This page offers a comparison of the features available for CircleCI Machine runner vs. CircleCI Container Runner.
+
+[#feature-comparison-matrix]
+== Comparison matrix
+
+[.table.table-striped]
+[cols=4*, options="header", stripes=even]
+|===
+|Feature
+|Description
+|Machine Runner
+|Container Runner
+
+|*Environment*
+|Where jobs run and how resources are managed
+|Full VM/bare metal
+|Kubernetes Pods
+
+|*Configuration File*
+|Primary file for runner setup
+|YAML config (location depends on OS)
+|Helm `values.yaml`
+
+|*Resource Control*
+|How compute resources are allocated
+|System level
```

Comment:
Wondering what system level means, can we expand a bit or link to a section that covers this?

Reply:
AFAIK there are ways to limit CPU/memory usage for given processes or services at the OS-level, but it's specific to each OS.